src/main/java/ch/njol/skript/effects/EffKill.java

```diff
@@ -44,24 +44,26 @@
 		"Note: This effect does not set the entity's health to 0 (which causes issues), but damages the entity by 100 times its maximum health."})
 @Examples({"kill the player",
 		"kill all creepers in the player's world",
-		"kill all endermen, witches and bats"})
+		"kill all endermen, witches and bats",
+		"kill the player ignoring totem of undying",
+		"kill target entity ignoring the totem",
+		"kill target player ignoring resurrection"})
 @Since("1.0")
 public class EffKill extends Effect {
 
 	static {
-		Skript.registerEffect(EffKill.class, "kill %entities%");
+		Skript.registerEffect(EffKill.class, "kill %entities%", "kill %entities% ignoring [the] (resurrection|revival|[undying] totem|totem [of undying])");
```

Comment:
I'm not a huge fan of the syntax. I might prefer to just keep it simple:
```
ignoring [the|any] totem[s] [of undying]
```
I'd like to see what others think too though

Reply:
What me and others have discussed on discord, is that the kill effect should be recoded to actually kill, since as of right now, it really just feels like a hardcoded damage effect. Another point is that the vanilla /kill command ignores the totem, whereas this, it does _not_ ignore the totem.

Reply:
Pickle is talking about the pattern being overly complex, not about it's behavior.

Reply:
personally i think `ignoring [the|any] totem[s] of undying` should be it, as `ignoring totems` sounds too ambiguous imo